js/CharModel.js

```diff
@@ -4,7 +4,7 @@ class CharModel{
         this.y = Y;
         this.energy = Energy;
         this.whiffles = Whiffles;
-        this.inventory;
+        this.inventory = new Array();
```

Comment:
Great! I know we'll need this soon.
I've only seen the Javascript this.inventory = []; syntax, not this "new Array". Neat